tests/Nox.Types.Tests/Types/Length/LengthTests.cs

```diff
@@ -170,27 +202,27 @@ public void Length_NonEquality_SpecifyingLengthUnit_WithDifferentUnit_Tests()
 
     private static void AssertAreEquivalent(Length expected, Length actual)
     {
-        Assert.Equal(expected, actual);
+        actual.Should().Be(expected);
 
-        Assert.True(expected.Equals(actual));
+        expected.Equals(actual).Should().BeTrue();
 
-        Assert.True(actual.Equals(expected));
+        actual.Equals(expected).Should().BeTrue();
 
-        Assert.True(expected == actual);
+        (expected == actual).Should().BeTrue();
 
-        Assert.False(expected != actual);
+        (expected != actual).Should().BeFalse();
     }
 
     private static void AssertAreNotEquivalent(Length expected, Length actual)
     {
-        Assert.NotEqual(expected, actual);
+        actual.Should().NotBe(expected);
 
-        Assert.False(expected.Equals(actual));
+        expected.Equals(actual).Should().BeFalse();
 
-        Assert.False(actual.Equals(expected));
+        actual.Equals(expected).Should().BeFalse();
 
-        Assert.False(expected == actual);
+        (expected == actual).Should().BeFalse();
 
-        Assert.True(expected != actual);
+        (expected != actual).Should().BeTrue();
```

Comment:
usually we do like this, for this and all previous (the first assert is ok):
```suggestion
        actual.Should().NotBe(expected);
```